include/mbedtls/ssl.h

```diff
@@ -801,6 +804,11 @@ typedef struct mbedtls_ssl_key_cert mbedtls_ssl_key_cert;
 typedef struct mbedtls_ssl_flight_item mbedtls_ssl_flight_item;
 #endif
 
+#if defined(MBEDTLS_SSL_EARLY_DATA) && defined(MBEDTLS_SSL_CLI_C)
+#define MBEDTLS_SSL_EARLY_DATA_NOT_SENT       0
+#define MBEDTLS_SSL_EARLY_DATA_REJECTED       1
+#define MBEDTLS_SSL_EARLY_DATA_ACCEPTED       2
```

Comment:
From https://github.com/Mbed-TLS/mbedtls/commit/7985ebac3d916ee95a7fbe48eeccc93dda99ed35#diff-0d03d729015f7890ce831f26a6ec581b99c8b0ab38bdb81982b59f47298636c9R4884
```suggestion
#define MBEDTLS_SSL_EARLY_DATA_STATUS_UNKNOWN   0
#define MBEDTLS_SSL_EARLY_DATA_STATUS_NOT_SENT  1
#define MBEDTLS_SSL_EARLY_DATA_STATUS_REJECTED  2
#define MBEDTLS_SSL_EARLY_DATA_STATUS_ACCEPTED  3
```